lib/node_modules/@stdlib/math/base/special/cfloorn/lib/native.js

```diff
@@ -0,0 +1,89 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var Complex128 = require( '@stdlib/complex/float64' );
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Rounds each component of a double-precision complex floating-point number to the nearest multiple of `10^n` toward negative infinity.
+*
+* @private
+* @param {Complex128} z - complex number
+* @param {Complex128} n - integer power of 10
+* @returns {Complex128} result
+*
+* @example
+* var Complex128 = require( '@stdlib/complex/float64' );
+* var real = require( '@stdlib/complex/real' );
+* var imag = require( '@stdlib/complex/imag' );
+*
+* // Round components to 2 decimal places:
+* var v = cfloorn( new Complex128( -3.141592653589793, 3.141592653589793 ), -2 );
+* // throws <ReferenceError>
+*
+* var re = real( v );
+* // throws <TypeError>
```

Comment:
@steff456 These all need to be fixed. The reason for the type errors was due to an incorrect variable name.